cloudbuild.yaml

```diff
@@ -1,31 +1,36 @@
-timeout: 20m
+##################################################################################
```

Comment:
Would be nice to follow up a with Travis or Cloudbuild verification to make sure that this file is up-to-date. Can serve as a check for new PRs.